.changeset/sour-spiders-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kadena/react-ui": patch
```

Comment:
Can you update this to `none`? Right now the package isn't published so we don't need to bump versions. Might also be good to discuss this as a team though